011_using_gohack/script.sh

```diff
@@ -68,6 +70,8 @@ go run .
 gohack undo rsc.io/quote
 go run .
 
+
 # block: version details
 go version
-echo "gohack commit $(cd $GOPATH/src/github.com/rogpeppe/gohack; git rev-parse HEAD)"
+cd /tmp/gohack
+echo "gohack commit $(git rev-parse HEAD)"
```

Comment:
What's `gohack commit` supposed to be doing here?

Reply:
It's just a way of manifesting the version we used whilst generating the guide: https://github.com/go-modules-by-example/index/blob/4284d09f20ccd914cf0ba780cfff4ee56337437e/011_using_gohack/README.md#version-details